package-lock.json

```diff
@@ -825,7 +825,7 @@
       "dependencies": {
         "chalk": {
           "version": "1.1.3",
-          "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
+          "resolved": "http://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```

Comment:
Looks like you're using an old version of npm and it's unnecessarily generating a diff here. On my local machine it's changing it back to https with `npm 6.4.1`.

Reply:
That's odd, because I deliberately updated npm right before making these changes.

```
+ npm@6.4.1
updated 1 package in 17.993s
```

I'll look into it.

Reply:
Checking out this branch again and running `npm install` flips this line back to https. Just some oddity with npm I guess. I'll amend the commit.